src/geo/lng_lat.js

```diff
@@ -2,6 +2,10 @@
 
 import {wrap} from '../util/util.js';
 import LngLatBounds from './lng_lat_bounds.js';
+import {mercatorZfromAltitude} from './mercator_coordinate.js';
+import EXTENT from '../data/extent.js';
+
+const DEG_TO_RAD = Math.PI / 180;
```

Comment:
This could use the helper from `uilt/` https://github.com/mapbox/mapbox-gl-js/blob/ad9c713939882f23db7902361d1d34a96280def6/src/util/util.js#L25